arm-resources/links/2016-09-01/swagger/links.json

```diff
@@ -0,0 +1,300 @@
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The filter to apply on the list resource links operation. The supported filter for list resource links is targetId."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceLinkResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/ResourceLinkFilter"
+      }
+    },
+    "/{scope}/providers/Microsoft.Resources/links": {
+      "get": {
+        "tags": [
+          "ResourceLinks"
+        ],
+        "operationId": "ResourceLinks_ListAtSourceScope",
+        "description": "Gets a list of resource links at and below the specified source scope. For example, to list resource links at and under a resource group, set the scope to /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myGroup.",
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The fully qualified Id of the source resource scope. For example, to list resource links at and under a resource group, set the scope to /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myGroup.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "enum": ["atScope()"],
+            "x-ms-enum": {
+              "name": "Filter",
+              "modelAsString": false
+            },
+            "type": "string",
+            "description": "The filter to apply on the list resource links operation. To list links only at the specified scope (not below the scope), use Filter.atScope()."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceLinkResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/ResourceLinkFilter"
+      }
+    }
+  },
+  "definitions": {
+    "ResourceLinkFilter": {
+      "properties": {
+        "targetId": {
+          "type": "string",
+          "description": "The target Id of the resource."
+        }
+      },
+      "description": "Resource link filter."
+    },
+    "ResourceLinkResult": {
+      "properties": {
+        "value": {
```

Comment:
Shouldn't this be a default for list result definitions? I'll add it here, but we'll need to make value required in our other specs as well, to keep it consistent